Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps-01-code-0004.swift

```diff
@@ -6,7 +6,7 @@ struct SyncUpsList {
   @ObservableState
   struct State: Equatable {
     @Presents var addSyncUp: SyncUpForm.State?
-    @Shared(.syncUps) var syncUps
+    @Shared(.fileStorage(.syncUps)) var syncUps: IdentifiedArrayOf<SyncUp> = []
```

Comment:
📓 This is not about going back to unsafe shared key but due to the insertion of a new step to explain the "withLock" changes before the step updating the code to use type safe shared key. It's easier to review these changes on the real compiled documentation 😉